src/components/text-and-media-snippet/text-and-media-snippet.jsx

```diff
@@ -17,9 +17,17 @@ const TextAndMediaSnippet = props => (
     <div className={classNames('text-and-media-snippet', props.className)}>
         <div className="half">
             <h4>{props.title}</h4>
-            <p>
-                {props.children}
-            </p>
+            {props.children.length > 0 ?
+                <div>
+                    {/* eslint-disable */}
+                    {props.children.map((paragraph, i) => { 
+                        return <p key={i}>{paragraph}</p>;
+                    })}
+                </div> :
```

Comment:
@seotts you are correct that last year's only had one paragraph. I've added the ternary clause here to account for snippets that only have one paragraph -- I checked the 2020 report locally and it still looks good with these changes.

Reply:
great, thanks!